src/Microsoft.ML.Transforms/UngroupTransform.cs

```diff
@@ -389,56 +424,11 @@ public int GetCommonPivotColumnSize()
                 }
                 return size;
             }
-
-            public int ColumnCount => _inputSchema.Count;
```

Comment:
>public int ColumnCount => _inputSchema.Count; [](start = 12, length = 45)

why not keep it?

Reply:
Just added it back. Please take a look at `InputColumnCount` in the next iteration.

---
In reply to: [246129129](https://github.com/dotnet/machinelearning/pull/2066#discussion_r246129129) [](ancestors = 246129129)